src/App.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { Route, Routes } from 'react-router-dom';
+import Header from './Components/Header';
+import Books from './Components/pages/Books';
+import Categories from './Components/pages/Categories';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div>
+        <Header />
+        <Routes>
+          <Route path="/" element={<Books />} />
+          <Route path="/categories" element={<Categories />} />
+        </Routes>
+      </div>
+    );
+  }
+}
```

Comment:
- [ ] **[OPTIONAL]:** Great job working on your app! 💯 I would suggest that you use Functional components from now on as they're the new way of writing react components and they provide very nice features like hooks. 🤓 

I would also want to point out that the latest version of react-router-dom suggests a different approach to handle the routing, at the moment this works, but If you're curious take a look at their [official tutorial](https://reactrouter.com/en/main/start/tutorial). 🤓 🔧